src/permissions/Permissions.service.ts

```diff
@@ -174,7 +180,7 @@ export class PermissionsService {
     clientIp: string,
     token: string,
     url: string,
-  ): Promise<FacilityAccessDTO[]> {
+  ): Promise<FacilityAccessWithCertStatementFlagDTO> {
```

Comment:
Since the return type has changed here also, make sure 
`return permissionResult.data;`
is returning data in the format of FacilityAccessWithCertStatementFlagDTO.

Reply:
Hi Yonatan. in https://github.com/US-EPA-CAMD/easey-content/blob/master/dev/auth/mockPermissions.json, the mocked permissions are still only contains userId and facilities. I was wondering if I should make changes to the file to adjust the structure and add my own testing account in it, and then submit PR for easey-content repo.
also, is that the only thing I need to adjust to cope with the changes we made in auth-api or there's more that I didn't aware of?